sycl/source/detail/force_device.cpp

```diff
@@ -21,18 +22,22 @@ bool match_types(const info::device_type &l, const info::device_type &r) {
 
 info::device_type get_forced_type() {
   if (const char *val = std::getenv("SYCL_DEVICE_TYPE")) {
-    if (string_class(val) == "CPU") {
+    std::string type(val);
+    std::transform(type.begin(), type.end(), type.begin(), ::tolower);
```

Comment:
Or `::toupper`? :) Just to "retain the spirit", I guess

Reply:
Obviously, I prefer "tolower()".